setup.py

```diff
@@ -48,24 +48,26 @@ def run(self):
 
 
 setup(
-    name="darglint",
-    version="1.8.1",
+    name="darglint2",
+    version="2.0.0",
```

Comment:
@9dogs do you think this rename and maintainer change warrants a major version bump?

Reply:
Good question. It's hard to tell how users interpret semver in this case. I personally think of it as "as long as `darglint[2]`s version is <2.0.0, the things I've read in the documentation like docstring formats, config parameters, etc. generally hold. If there is a bump in the major version, I probably need to re-check the changelog/documentation". 

So despite "breaking changes" (and since they are not in the API/protocols plane), I would argue for keeping the versioning continuous.

Reply:
Thanks for your well-informed opinion. I changed to a patch version bump (1.8.2).